packages/bot-web-ui/src/assets/sass/save-modal.scss

```diff
@@ -131,17 +131,23 @@ div.radio-group {
             flex-direction: column;
             justify-content: space-between;
         }
-        & .dc-page-overlay__header {
-            position: fixed;
-            top: 4rem;
-            width: 100%;
-            height: 4rem;
-        }
-        & .dc-page-overlay__content {
-            margin-top: 40px;
+        & .dc-page-overlay {
+            margin-top: -56px;
+
+            &__header {
+                position: fixed;
+                top: 4rem;
+                width: 100%;
+                height: 4rem;
+            }
+            &__content {
```

Comment:
@aaron-binary 
save modal changes are not related to the scope of this card but as this changes are to small to solve some inconsistencies with design which happened after update toolbox on responsive size I've added here
![image](https://user-images.githubusercontent.com/60640951/87045806-8f22b000-c20d-11ea-9296-3cf1d941d9e4.png)